static/sass/styles.scss

```diff
@@ -405,6 +405,17 @@ summary {
   margin-bottom: 0;
 }
 
+.p-table--advantage-contact {
+  @media only screen and (min-width: $breakpoint-large) {
+    width: 66%;
+  }
+
+  th,
+  td {
+    flex: 1 !important;
```

Comment:
Are tables flex?

Reply:
`p-table--expanding` are flex, and `td`s under the `p-table--advantage ` class have been given various flex values - the parent table uses both of these classes.